modules/post/windows/gather/credentials/securecrt.rb

```diff
@@ -170,9 +171,16 @@ def securecrt_store_config(config)
 
   def run
     print_status("Gathering SecureCRT session information from #{sysinfo['Computer']}")
-    parent_key = 'HKEY_CURRENT_USER\\Software\\VanDyke\\SecureCRT'
-    # get session file path
-    securecrt_path = expand_path(registry_getvaldata(parent_key, 'Config Path') + session.fs.file.separator + 'Sessions')
+    securecrt_path = ''
+    if datastore['SESSION_PATH'].to_s.empty?
+      parent_key = 'HKEY_CURRENT_USER\\Software\\VanDyke\\SecureCRT'
+      # get session file path
+      root_path = registry_getvaldata(parent_key, 'Config Path')
+      securecrt_path = expand_path(root_path + session.fs.file.separator + 'Sessions') if !root_path.nil?
```

Comment:
So here is my thought, its possible for us to change this to `!root_path.to_s.empty?`. My thought is that its possible for `registry_getvaldata` to potentially return an empty string, and whilst highly unlikely, it is possible given the volatility of the Windows registry itself, so we should try an account for this the best we can.